docs/src/modules/spring/pages/spring-boot-integration.adoc

```diff
@@ -0,0 +1,32 @@
+= Spring Boot Integration
+
+The Kalix Spring SDK provides a familiar developer experience for Java developers that are used to Spring. Its `Main`
+class is annotated with `@SpringBootApplication` and starts exactly the same way as in a Spring Boot application, namely
+by calling `SpringApplication.run(Main.class, args)`.
+
+For all matters, it's a Spring Boot application that runs and connects with the Kalix Proxy. As such, your application can benefit of most features available to Spring applications. However, some features are not available or are not recommended when building a Kalix application based on the Spring SDK. Those are explained in the sections below.
```

Comment:
Maybe link "Kalix Proxy" to https://docs.kalix.io/reference/glossary.html#proxy ? Might be useful for this first reference.

Reply:
Will do